saas/axops/src/applatix.io/axops/configuration/schema.go

```diff
@@ -4,8 +4,9 @@ import "applatix.io/axdb"
 
 const (
 	ConfigurationTableName   = "configuration"
-	ConfigurationUserName    = "user"
+	ConfigurationUserName    = "user" // will use this as namespace
 	ConfigurationName        = "name"
+	ConfigurationIsSecrets   = "is_secret"
```

Comment:
how is schema update handled by axdb? For existing configs, will this field be set to false?